discord_slash/client.py

```diff
@@ -417,12 +415,19 @@ def add_subcommand(self,
         name = name or cmd.__name__
         name = name.lower()
         description = description or getdoc(cmd)
-        guild_ids = guild_ids if guild_ids else []
-
-        if base in self.commands:
-            for x in guild_ids:
-                if x not in self.commands[base].allowed_guild_ids:
-                    self.commands[base].allowed_guild_ids.append(x)
+        guild_ids = guild_ids if guild_ids else ["global"]
+
+        for i in range(len(guild_ids)):
```

Comment:
```suggestion
        for guild in guild_ids:
```

Reply:
same as before
I chose to use `i in range` as I'm updating the list of which I'm iterating over